packages/labs/virtualizer/src/Virtualizer.ts

```diff
@@ -550,8 +553,16 @@ export class Virtualizer {
   }
 
   _updateLayout() {
-    if (this._layout) {
-      this._layout!.items = this._items;
+    // Only update the layout and trigger a re-render if we have:
+    //   a) A layout
+    //   b) A scrollerController, which means we're connected
+    //   c) An offsetParent, which means we're not hidden
```

Comment:
This will be `null` if the hostElement has a parent that is positioned `fixed`. The virtualizer doesn't render inside a `mwc-dialog` anymore now for example.